### PR TITLE
Sidebar implementation

### PR DESCRIPTION
**Pull request by Davidslv**
Trello card: https://trello.com/c/riC1fX76/152-design-build-a-sidebar-that-includes-steps-and-content-links-highlighting-the-page-youre-on

Besides the following screenshot, the navigation link ("Learning to drive: step by step") is also sticky after scrolling down.

screen shot 2017-09-04 at 16 25 20

### CODE REVIEW

File: app/views/content_items/_sticky_nav.html.erb [hunk 1]
```diff
@@ -5,3 +5,14 @@
     </div>
   </div>
 </div>
+
+<script>
```
Comment (sihugh) [inline app/views/content_items/_sticky_nav.html.erb]: We might be better off using https://github.com/alphagov/govuk_frontend_toolkit/blob/master/docs/javascript.md#stick-at-top-when-scrolling for the sticky nav unless there's a particular reason why not?

File: app/assets/javascripts/accordion-with-descriptions.js [hunk 2]
```diff
@@ -0,0 +1,327 @@
+// Most of this is taken from the service manual and at some point needs to be
```
Comment (sihugh) [inline app/assets/javascripts/accordion-with-descriptions.js]: Is this just lifted and shifted, or did you have to change anything?

File: app/views/content_items/_sticky_nav.html.erb (hunk 1 above)
Comment (Davidslv) [reply app/views/content_items/_sticky_nav.html.erb]: That's a good shout @sihugh, we decided to write our own implementation for educational purposes, but we can see the benefit of use that instead and we will change it afterwards (another PR).

File: app/assets/javascripts/accordion-with-descriptions.js (hunk 2 above)
Comment (Davidslv) [reply app/assets/javascripts/accordion-with-descriptions.js]: My initial thought is that this file was going to be needed to implement the sidebar functionality. I'm still not sure if the frontend developer will need it, but this makes it easier to have it on the prototype if needed.

**Comment by Davidslv** [comment]
Merging this for frontend developers to continue and fork from master branch.

**Pull request merged**